packages/tinymist/package.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: tinymist
+description: |
+  Tinymist [ˈtaɪni mɪst] is an integrated language service for Typst [taɪpst]. You can also call it "微霭" [wēi ǎi] in Chinese
+homepage: https://github.com/Myriad-Dreamin/tinymist
+licenses:
+  - Apache-2.0
+languages:
+  - Typst
+categories:
+  - LSP
+
+source:
+  id: pkg:github/Myriad-Dreamin/tinymist@v0.10.2
```

Comment:
There already is a newer version available, would advise to update it

Reply:
Wouldn't the bot automatically update it anyway?

Reply:
Wouldn't the bot automatically update it anyway?